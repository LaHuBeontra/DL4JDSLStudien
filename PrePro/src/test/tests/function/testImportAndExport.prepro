function main() {
	import vec3 p3, vec3 p1, mat4 testMat4_1, const quaternion_1;

	export p1, p3, testMat4_1, quaternion_1;
}
