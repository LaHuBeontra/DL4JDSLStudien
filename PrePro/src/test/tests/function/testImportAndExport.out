DefaultPreProDataSet{
dataSetEntries=
[DataSetEntry{variableName='p1', variableShape=[4, 3], startColumn=0, length=3}, DataSetEntry{variableName='p3', variableShape=[4, 3], startColumn=3, length=3}],
dataSet=
[[42.00,  0.00,  0.00,  42.00,  0.00,  1.00],  
 [1.00,  0.00,  0.00,  1.00,  0.00,  1.00],  
 [5.00,  0.00,  0.00,  5.00,  0.00,  1.00],  
 [6.00,  0.00,  0.00,  6.00,  0.00,  1.00]]

calculated_variables=
p1=[[42.00,  0.00,  0.00],  
 [1.00,  0.00,  0.00],  
 [5.00,  0.00,  0.00],  
 [6.00,  0.00,  0.00]]
p3=[[42.00,  0.00,  1.00],  
 [1.00,  0.00,  1.00],  
 [5.00,  0.00,  1.00],  
 [6.00,  0.00,  1.00]]
}